docs/src/main/paradox/release-notes/1.0-M1.md

```diff
@@ -0,0 +1,57 @@
+# Alpakka Kafka 1.0-M1
+
+## First milestone release for Alpakka Kafka 1.0
+
+On this [Road to Alpakka 1.0](https://akka.io/blog/news/2018/08/30/alpakka-towards-1.0) we may introduce non-compatible API changes - **but did not in Alpakka Kafka 1.0-M1 vs. 0.22**. From version 1.0 Alpakka will stay [binary-compatible](https://doc.akka.io/docs/akka/current/common/binary-compatibility-rules.html#binary-compatibility-rules) between minor releases.
+
+Having that said, Alpakka will start to make use of the `@ApiMayChange` annotation to keep the door open for API changes in new modules, so that they can evolve more rapidly than other modules within Alpakka.
```

Comment:
```suggestion
Having that said, Alpakka Kafka will start to make use of the `@ApiMayChange` annotation to keep the door open for API changes, so that new feature API can evolve more rapidly than other parts of Alpakka Kafka.
```